lib/acts_as_rdfable/acts_as_rdfable_core.rb

```diff
@@ -6,12 +6,21 @@ def acts_as_rdfable(&configuration_block)
       raise InvalidClassError unless self < ActiveRecord::Base
 
       define_method :rdf_annotations do
-        RdfAnnotation.for_table(self.table_name)
+        self.class.rdf_annotations
       end
 
       define_method :rdf_annotation_for_attr do |attr|
+        self.class.rdf_annotations.where column: attr
```

Comment:
This should probably just call the class version of the method, so that if we change the scope `for_table_column` we don't have to know/remember to also change the duplicated logic here?

Reply:
You are right. Changed this issue on the last commit.